experimental/entity/entity.go

```diff
@@ -0,0 +1,56 @@
+package entity
+
+// This is the base class for (non-raw) kinds
+type Envelope struct {
```

Comment:
This "envelope" type is what i'm pointing at with "standardized object metadata" in [this section](https://docs.google.com/document/d/12vYU-DRlS0yYxvXW52cOH7kSIxmJAER_4_mlA0An_QM/edit#heading=h.7udkiq1qxut3)

Reply:
👍 yes some sort of main object that most thigns derive from